examples/off_limb_enhance.py

```diff
@@ -0,0 +1,73 @@
+"""
+===========================
+Enhancing Off-limb emission
+===========================
+
+This example shows how to enhance emission above the limb.
+"""
+import numpy as np
+import astropy.units as u
+import matplotlib.pyplot as plt
+import matplotlib.colors as colors
+
+from sunpy.data.sample import AIA_171_IMAGE
```

Comment:
this line break is inconsistent with the other import blocks, personally I like line breaks in imports, but consistency is important.